guides/screen-reader-differences.md

```diff
@@ -0,0 +1,126 @@
+
+
+Headings
+- 1: H1
+- 2: H2
+- 3: H3
+- 3: H3
+
+![screen shot 2018-06-28 at 11 31 26 am](https://user-images.githubusercontent.com/704760/42045575-d944241c-7ac9-11e8-9473-ce2b2726cac7.png)
+
+
+
+Because hidden elements are not considered part of the page content for screen readers*, they are not included in the elements listed by the rotor menus. This has important implications for how we design and code interactive elements on our webpages. For example, in the current implementation of our global navigation "mega menu", none of the content that's in the mega menu dropdown is accessible to the VoiceOver rotor unless the user has opened the dropdown with a mouse hover or the keyboard. This as a general problem with drop down menus and similar interaction designs that affects VoiceOver users (and likely all screen reader users- we just haven't tested other software yet).
+
+*This is not the case for elements that are visually hidden using the same CSS techniques as are used to visually hide the "Skip nav" link on every page. It only applies to techniques that hide the content in a straightforward manner such as using `display: none` or `visibility: hidden`.
+
+#### 3. Forms mode
+VoiceOver does not have a forms or application mode. JAWS and NVDA do.
+
+"Since screen readers use many of the keys on the keyboard for quick navigation, trying to enter text in a form field presents an interesting problem. For example, trying to type the letter "h" in browse mode with JAWS would take you out of the form field and navigate to the next heading on the page. Luckily, screen readers have a built-in fix for this issue: Forms mode. In forms mode, most standard screen reader quick keys are deactivated, so that you can fill out forms without accidentally triggering a quick key." [Source](https://webaim.org/articles/jaws/#formsmode)
+
+Read about the specific [JAWS keyboard commands that apply in forms mode](https://webaim.org/articles/jaws/#formsmode) and the [NVDA forms mode](https://webaim.org/articles/nvda/#formsmode).
+
+#### 4. Application mode
+VoiceOver does not have a forms or application mode; JAWS and NVDA do. The application mode is triggered by the use of the `role="application"` ARIA attribute, which disables the default keyboard shortcuts for navigating the page. The developer is then responsible for implementing unique interaction shortcuts with JavaScript.
+
+The W3C recommends developers **not use `role="application"`** if the interface is made up of standard HTML elements such as form elements, links, headings, etc.
+
+"You only want to use `role=application` if the content you’re providing consists of only focusable, interactive controls, and of those, mostly advanced widgets that emulate a real desktop application" ([source](https://www.w3.org/TR/using-aria/#using-application)). This should virtually never be the case for a website that we are developing.
+
```

Comment:
This application mode info is duplicated below. Could it be consolidated into the "Lessons" section?

Reply:
👍